### PR TITLE
Pull Request for OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by chuaindia**
In this branch following facilities are incorporated :

- A "app. rb" file is created which has the following methods: ---
      Listing of all books.
      Listing of all people.
      Creation of a person (teacher or student, not a plain Person).
      Creation of a book.
      Creation of a rental.
      Listing of all rentals for a given person's id.

- The "main. rb" file will have the following facilities: -----
     Present the user with a list of options to perform.
     Lets users choose an option.
     If needed, ask for parameters for the option.
     Have a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    puts 'Welcome to the Library'
+    puts 'Please enter a number to select the option of your choice from the following list'
+
+    @list = {
+      '1' => 'List of All Books',
+      '2' => 'List of All Peoples',
+      '3' => 'Create a Person',
+      '4' => 'Create a Book',
+      '5' => 'Create a Rental',
+      '6' => 'List of All Rentals for a person ID',
```
Comment (1GORDON) [inline main.rb:14]: Great job @chuaindia 🤝

- [x] You may kindly fix the listing of rentals. option 6. When I try to list rentals, I get the error below.
error

File: README.md [hunk 2]
```diff
@@ -1 +1,274 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
```
Comment (1GORDON) [inline README.md]: Great job @chuaindia 🤝

- [x] [Optional] You may remove the live demo Link from your readme if the project is not deployed. This makes your readme file more professional.

File: README.md [hunk 3]
```diff
@@ -1 +1,274 @@
...
+
+  - ["<" sign. on the left hand side the derived class name will appear and the parent classname will be placed on the right hand side.]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (1GORDON) [inline README.md:263]: Great job @chuaindia 🤝

- [x] [Optional] Kindly fix the broken MIT license link to make your readme file more professional.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @chuaindia ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (chuaindia) [reply README.md]: Done.

File: main.rb (hunk 1 above)
Comment (chuaindia) [reply main.rb:14]: Done accordingly.

**Review by AnselemOdims — APPROVED** [review]
### Hi  @chuaindia 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**